contracts/margin/external/ERC20/ERC20Position.sol

```diff
@@ -153,16 +149,14 @@ contract ERC20Position is
         address margin,
         address initialTokenHolder,
         address[] trustedRecipients,
-        address[] trustedWithdrawers,
-        string _symbol
+        address[] trustedWithdrawers
     )
         public
         OnlyMargin(margin)
     {
         POSITION_ID = positionId;
         state = State.UNINITIALIZED;
         INITIAL_TOKEN_HOLDER = initialTokenHolder;
-        symbol = _symbol;
```

Comment:
this should save some deploy gas too 😄

Reply:
Yeah I'm still not sure what the gas cost is per byte of code

Reply:
well it does an SSTORE so likely 20k

Reply:
think it stores that variable in storage, not code